content/en/tracing/trace_explorer/visualize.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Span Visualizations
+kind: documentation
+description: 'View spans in a list, or aggregate spans into timeseries, top lists and more.'
+further_reading:
+    - link: 'tracing/trace_explorer/'
+      tag: 'Documentation'
+      text: 'Trace Explorer'
+---
+
+## Overview
+
+Visualizations define how the queried span data is displayed. Select relevant visualizations to surface valuable information, such as a **list** for individual events, or as **timeseries** or **top lists** for aggregates. 
+
+## List view
+
+The list view displays a list of spans that match the selected context, defined by the [search bar query][1] filter and a [time range][2].
+
+In the **Show** row, choose which information of interest to display as columns. Manage the columns by either:
```

Comment:
Previously, this said "In the choose which", so please make sure that I added an appropriate wording. :)